app/routes/api/contact-history.js

```diff
@@ -33,7 +33,7 @@ module.exports = [
         payload: Joi.object({
           user: Joi.string(),
           farmerName: Joi.string(),
-          orgEmail: Joi.string(),
+          orgEmail: Joi.string().allow(null),
```

Comment:
Worth adding a unit test in the test file for this? Looking at what is there currently there isn't one that covers the case orgEmail is sent in with a null value, so for belt and braces maybe nice to add a case that proves the functionality works as expected still when a null gets sent through?

Also, whilst you're there if you could change the title of the test in there currently called
`test('should through error if the sbi is not send'` , it would remove us of this abomination of usage of the English language once and for all, and would make me feel better about life

Reply:
Test for null orgEmail added. https://github.com/DEFRA/ffc-ahwr-application/pull/280/files#diff-70d26fc46f0610aee7c76512ad065a7f6821ea46116e2e183284560fc9f9d7abR108